django/db/models/sql/compiler.py

```diff
@@ -701,7 +701,7 @@ def get_from_clause(self):
         """
         result = []
         params = []
-        for alias in self.query.alias_map:
+        for alias in tuple(self.query.alias_map):
```

Comment:
I'm not sure if this change is needed.

Reply:
because we are mutating self.query.alias_map during iteration.

```
RuntimeError: OrderedDict mutated during iteration
```